### PR TITLE
Fix/activity card desc

### PR DESCRIPTION
**Pull request by zahirekrem09**
#125

### CODE REVIEW

File: src/components/shared/card.js [hunk 1]
```diff
@@ -34,8 +38,8 @@ export const Card = props => {
         >
           {title}
         </Heading>
-        <Text fontSize='md' lineHeight='base'>
-          {description}
+        <Text fontSize='md' lineHeight='base' noOfLines={3}>
+          {description?.split('.')[0]}.
         </Text>
```
Comment (7alip) [inline src/components/shared/card.js:43]: I think we don't need to change the card component itself. We actually need to fix it in the backend by providing `description` field as a summary of the context. I was expecting to truncate the content and pass it to `description` prop of the Card.

https://github.com/wsvvrijheid/wsvvrijheid/blob/e360d381aafab81a79696cb5075a02ae7d48f056/src/pages/activities/index.js#L17-L24

`description={activity.content.split(.)}`

But it's also not a good solution as we might have some markdown formatting characters (# * ~ _) in the text. We tried to fix it in another project:

https://github.com/wsvvrijheid/samenvvv/blob/2f5ece9f355ed2b131cc4ae2c081d37893bd6e26/src/utils/makeTwitterContent/index.ts

But my suggestion is that:

Fix the problem in the backend (provide description field of activity). Pass this value to the card.

Shall we keep this PR open until we fix it in the backend?

**Review by 7alip — COMMENTED** [review]
Thank you for the submission. I left a note and I recommend to solve it in the backend.

File: src/components/shared/card.js (hunk 1 above)
Comment (7alip) [reply src/components/shared/card.js:43]: Art model has `description` field now. You can try it out @zahirekrem09

**Review by 7alip — CHANGES_REQUESTED** [review]

File: src/components/shared/card.js (hunk 1 above)
Comment (zahirekrem09) [reply src/components/shared/card.js:43]: Updated with the `description` from the backend

**Review by MustafaTRKYLMZ — APPROVED** [review]
Thanks a lot

File: src/theme/global.js [hunk 2]
```diff
@@ -8,5 +8,10 @@ export const styles = {
       fontSize: { base: 'md', lg: 'lg' },
       lineHeight: 1.5,
     },
+    iframe: {
+      border: 'none',
+      maxWidth: '100%',
+      maxHeight: '450px',
```
Comment (7alip) [inline src/theme/global.js:14]:
```suggestion
      width: '100%',
      height: '450px',
```

I think this will fix the issue

**Review by 7alip — CHANGES_REQUESTED** [review]

**Review by 7alip — APPROVED** [review]
Thank you! You can merge it

**Pull request merged**